drools-wb-webapp/src/main/java/org/drools/workbench/client/home/HomeProducer.java

```diff
@@ -36,21 +36,14 @@
 /**
  * Producer method for the Home Page content
  */
-@ApplicationScoped
+@Dependent
 public class HomeProducer {
 
-    private HomeModel model;
-
-    @Inject
-    private PlaceManager placeManager;
-
-    @Inject
-    protected LibraryMonitor libraryMonitor;
```

Comment:
@mbarkley are you sure that this is not needed? Sometimes I saw on our codebase this Injection points used to "force" the bean startup.

Maybe is not on this case because this code lives on drools-wb-webapp, but we should be careful doing this.